pylabrobot/tilting/hamilton.py

```diff
@@ -1,18 +1,50 @@
+from pylabrobot.resources.coordinate import Coordinate
 from pylabrobot.tilting.tilter import Tilter
 from pylabrobot.tilting.hamilton_backend import HamiltonTiltModuleBackend
 
 
-def HamiltonTiltModule(  # pylint: disable=invalid-name
-  name: str,
-  com_port: str,
-  write_timeout: float = 10, timeout: float = 10
-) -> Tilter:
-  return Tilter(
-    name=name,
-    backend=HamiltonTiltModuleBackend(
-      com_port=com_port, write_timeout=write_timeout, timeout=timeout
-    ),
-    size_x=132,
-    size_y=92.57,
-    size_z=85.81,
-  )
+class HamiltonTiltModule(Tilter):
+  """ A Hamilton tilt module. """
+
+  def __init__(
+    self,
+    name: str,
+    com_port: str,
+    child_resource_location: Coordinate = Coordinate(1.0, 3.0, 83.55),
+    pedestal_size_z: float = 3.47,
+    write_timeout: float = 3,
+    timeout: float = 3,
+  ):
+    """ Initialize a Hamilton tilt module.
+
+    Args:
+      child_resource_location: The location of the child resource.
+      pedestal_size_z: The size of the pedestal in the z dimension.
+      hinge_coordinate: The coordinate of the hinge.
+      com_port: The communication port.
+      write_timeout: The write timeout. Defaults to 3.
+      timeout: The timeout. Defaults to 3.
+    """
+
+    super().__init__(
+      name=name,
+      size_x=132,
+      size_y=92.57,
+      size_z=85.81,
+      backend=HamiltonTiltModuleBackend(
+        com_port=com_port,
+        write_timeout=write_timeout,
+        timeout=timeout),
+      hinge_coordinate=Coordinate(6.18, 0, 72.85),
+      child_resource_location=child_resource_location,
+      # sites=create_homogeneous_carrier_sites(
+      #   klass=CarrierSite,
+      #   locations=[Coordinate(1.0, 3.0, 83.55)],
+      #   site_size_x=127.0,
+      #   site_size_y=86.0,
+      # ),
```

Comment:
is this still used? I don't think it was even before the commit I just made.

Reply:
Nope, forgot to delete - thanks